### PR TITLE
updates to leavetrack

### PR DESCRIPTION
**Pull request by barrypitman**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,215 +1,3 @@
...
-# Unit test / coverage reports
-.coverage
-.tox
-
-#Translations
-*.mo
-
-#Mr Developer
-.mr.developer.cfg
+.idea
```
Comment (barrypitman) [inline .gitignore]: The "gitignore" file is quite easy to use, and we don't need to ignore a lot of files. The most important files that we don't want going into version control are the IDEA project files (in the .idea folder & *.iml files), and the build/compiled files, in the "target" directory.

File: pom.xml [hunk 2]
```diff
@@ -55,7 +55,7 @@
         <dependency>
             <groupId>junit</groupId>
             <artifactId>junit</artifactId>
-            <version>4.8.2</version>
```
Comment (barrypitman) [inline pom.xml]: I got some errors when I tried to run "AppTests" from IDEA or using maven. It was mostly due to some version conflicts which should be fixed now.

File: pom.xml [hunk 3]
```diff
@@ -198,6 +203,26 @@
...
+                    </webAppConfig>
+                    <reload>manual</reload>
+                    <connectors>
+                        <connector implementation="org.eclipse.jetty.server.nio.SelectChannelConnector">
+                            <port>8080</port>
+                            <maxIdleTime>60000</maxIdleTime>
+                        </connector>
+                    </connectors>
+                </configuration>
+            </plugin>
```
Comment (barrypitman) [inline pom.xml]: The maven jetty plugin is what we normally use to run the webapps locally. To use it, run "mvn jetty:run" from the command line, which will start the server. Then, changes to JSPs and flow files we reflect automatically. Changes to java files will require a restart (Ctrl+C && mvn jetty:run)

File: src/main/webapp/WEB-INF/flows/changePassword/changePassword.jsp [hunk 4]
```diff
@@ -20,7 +20,7 @@
 
 <head>
     <title></title>
-    <link href="${pageContext.servletContext.contextPath}/resources/theme/css/design.css" rel="stylesheet"/>
+    <link href="<c:url value="/resources/theme/css/design.css"/>" rel="stylesheet"/>
```
Comment (barrypitman) [inline src/main/webapp/WEB-INF/flows/changePassword/changePassword.jsp]: using c:url tags will prepend the context path to the URL, same as ${pageContext.servletContext.contextPath}, its just a bit shorter :)

File: src/test/java/com/ubiquitech/app/AppTests.java [hunk 5]
```diff
@@ -17,7 +17,7 @@
 
 @RunWith(SpringJUnit4ClassRunner.class)
 @WebAppConfiguration
-@ContextConfiguration("file:../../../../../main/webapp/WEB-INF/spring-servlet.xml")
+@ContextConfiguration("file:src/main/webapp/WEB-INF/spring-servlet.xml")
 public class AppTests {
```
Comment (barrypitman) [inline src/test/java/com/ubiquitech/app/AppTests.java]: You should be able to run this test in IDEA (right click then "Run AppTests") and it will pass

Comment (Vane101) [inline src/test/java/com/ubiquitech/app/AppTests.java]: Thanks

**Pull request merged**